docs/limitations.md

```diff
@@ -242,12 +242,15 @@ See more documentation at
 
 #### `docker --device`
 
-host device support (`docker run --device`) is not supported. This could
-be mapped to the quite similar QEMU `-device` option to enact device
-pass-through if that particular device is supported by VFIO, but the QEMU
-option requires precise knowledge of which sort of device is being
-mapped, and that information is not necessarily easily available from
-the docker `--device` command.
+Support has been added to pass devices using [VFIO](https://www.kernel.org/doc/Documentation/vfio.txt) passthrough.
+Devices that support IOMMU can be assigned to `vfio-pci` driver and passed
+on the docker command line with --device=/dev/vfio/$(iommu_group_number),  
+where iommu_group_number is the IOMMU group that the device belongs to.
+If multiple devices belong to the same IOMMU group, then all of them will be 
+assigned to the Clear Containers VM.
+
+Support for passing other devices including block devices with --device
+has not been added yet.
```

Comment:
* use literal text for `--device`
* s/has not been added yet/is not yet available.